plexapi/__init__.py

```diff
@@ -21,9 +21,9 @@
 X_PLEX_CONTAINER_SIZE = CONFIG.get('plexapi.container_size', 100, int)
 X_PLEX_ENABLE_FAST_CONNECT = CONFIG.get('plexapi.enable_fast_connect', False, bool)
 
-# Plex Header Configuation
+# Plex Header Configuration
 X_PLEX_PROVIDES = CONFIG.get('header.provides', 'controller')
-X_PLEX_PLATFORM = CONFIG.get('header.platform', CONFIG.get('header.platorm', uname()[0]))
+X_PLEX_PLATFORM = CONFIG.get('header.platform', CONFIG.get('header.platform', uname()[0]))
```

Comment:
this could be described as breaking change.

anyway, since It was already merged and released, removing the second read:
- https://github.com/pkkid/python-plexapi/pull/1080

Reply:
My mistake... I should have realized that was a fallback.